assembl/scripts/ini_files.py

```diff
@@ -223,12 +223,10 @@ def populate_random(random_file, random_template=None, saml_info=None):
     from os import urandom
     from assembl.auth.make_saml import make_saml_key, make_saml_cert
     base = Parser()
-    if random_template is None:
-        random_template = os.path.basename(random_file) + ".tmpl"
```

Comment:
now that there are many templates, there is no explicit one. Explicit is better than implicit etc.